src/components/price/service.ts

```diff
@@ -59,21 +72,17 @@ export class PriceService {
         try {
             const pairPrice = this.getPairPrice(swap.network, swap.outputNetwork);
 
-            const inputDecimals = Config[swap.network].decimals;
-            const outputDecimals = Config[swap.outputNetwork].decimals;
+            const inputDecimals = this.blockchainConfig[swap.network].decimals;
+            const outputDecimals = this.blockchainConfig[swap.outputNetwork].decimals;
 
             const priceInBig = mulDecimals(pairPrice, outputDecimals);
 
             const outputAmount = mul(priceInBig, divDecimals(swap.inputAmount, inputDecimals));
 
-            const requestedAmount = mul(
-                outputAmount,
-                add(
-                    1,
-                    AppConfig.PRICE.TOLERANCE[swap.network + '-' + swap.outputNetwork] ||
-                        AppConfig.PRICE.TOLERANCE.DEFAULT
-                )
-            );
+            const pairSlippage = this.userConfig.PAIRS[`${swap.network}-${swap.outputNetwork}`].SLIPPAGE || 0;
+
+            const requestedAmount = mul(outputAmount, add(1, pairSlippage));
+
             const outputAmountWithoutFee = sub(swap.outputAmount, mul(swap.outputAmount, AppConfig.FEE));
```

Comment:
Both AppConfig and UserConfig have property named FEE. This is frustrating

Reply:
The user config does not have a property named FEE. However every pair has a FEE property. The difference is obvious.